py/src/braintrust/audit.py

```diff
@@ -0,0 +1,17 @@
+"""
+Utilities for working with audit headers.
+"""
+import base64
+import gzip
+import json
+from typing import List, TypedDict
+
+
+class AuditResource(TypedDict):
+    type: str
+    id: str
+    name: str
+
+
+def parse_audit_resources(marshaled: str) -> List[AuditResource]:
+    return json.loads(gzip.decompress(base64.b64decode(marshaled)))
```

Comment:
Does it make sense to have the "marshalled" payload be something slightly more elaborate so we can cleanly evolve it in the future? Like maybe it's a JSON array consisting of a version counter and a payload. Or if we wanted to be more fancy at the cost of a few more bytes, a metadata object and a payload.

Reply:
Done!